functions

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+# @brief dreport plugins functions.
+#
+# This file is part of klish project.
```

Comment:
I don't think this file is a parth of klish project anymore after porting it into obmc-yadro-cli.

Reply:
fixed